pkg/prometheus/promcfg.go

```diff
@@ -740,10 +743,11 @@ func (cg *configGenerator) generateProbeConfig(
 		}...)
 
 		cfg = append(cfg, yaml.MapItem{Key: "relabel_configs", Value: enforceNamespaceLabel(relabelings, m.Namespace, enforcedNamespaceLabel)})
-	}
-
-	// Generate kubernetes_sd_config section for ingress resources.
-	if m.Spec.Targets.StaticConfig == nil {
+	} else {
```

Comment:
I think there is nothing happening after the `if` block. So how about simply returning `return cfg, nil` after line 745 which alles unindenting the whole `else` block?

Reply:
Thanks, changed it accordingly